website/docs/riverpod_for_provider_users.mdx

```diff
@@ -18,62 +18,179 @@ import {
 This article is designed for people familiar with the [Provider] package who
 wants to learn about Riverpod.
 
-## The relationship between Riverpod and Provider
+Since Provider is widely popular, why would one migrate to Riverpod?
+
+## Provider's limitations
```

Comment:
Docusaurus already enables writing "articles". I've never had the need for this. But this could be a good use-case

Reply:
I've wrote all of these limitations in response of #2629 (see bullet points). I interpreted those bullet points as "necessary" to fix #2629. Moving "Provider's limitations" into a separate page makes sense to me

Reply:
I know. But keep in mind that #2676 is only a quick draft of what should be covered.  
In the end it's quite important to make the page readable. Otherwise people will simply not read it, because they are scared of walls of texts.

I think that's generally Riverpod's issue. There's too much docs :P

Reply:
I realize this. I'm sure I can be more concise, but at the same time I wrote these subparagraphs while reminding of all the questions we're given into discord daily.

So you're telling me that on one hand folks want documentation, and on the other they don't want to read it 😆 While this is true, it all boils down to split this page as suggested below.

ALSO some examples (code) might alleviate readability. What do you think?

Reply:
The content is great, don't worry about that. I'm mainly worried about formatting, which is not something those GitHub issues care about.

Examples would for sure help. Although it could also have the adverse effect (since the page would become an even larger wall of text).

I'd combine examples with your proposal of splitting this into subpages